src/server/handlers/schema.js

```diff
@@ -18,6 +18,6 @@ export const repo = new schema.Entity('repos', {
 export const repoList = new schema.Array(repo);
 
 export const snap = new schema.Entity('snaps', {}, {
-  idAttribute: 'name'
+  idAttribute: 'git_repository_url'
```

Comment:
I wonder if it is safe to assume that there is going to be only one snap with given repository URL in LP.

Especially in case of organisation repos, that can accessed and possibly built by multiple users.

@cjwatson Is it ok to use GH repository URL as a snap identifier on BSI? Is there only one snap for given repo or can there be multiple in some cases?